### PR TITLE
feat : Seacrh and filter with categories

### PR DESCRIPTION
**Pull request by anuj-kumary**
This PR includes:-
 - Search with name functionality
 - Filter the video based on category
 
Preview:- https://deploy-preview-7--techhub-react.netlify.app/

### CODE REVIEW

**Review by muskaanshah — APPROVED** [review]
Functionality working as expected, clean understandable code

File: src/Services/Services.js [hunk 1]
```diff
@@ -123,3 +125,18 @@ export const clearAllHistoryServices = async (encodedToken) =>
       authorization: encodedToken,
     },
   });
+
+export const searchVideos = (videos, search) => {
+  return search
+    ? [...videos].filter((video) =>
```
Comment (varnita619) [inline src/Services/Services.js:131]: good use of ternary operator

**Review by varnita619 — APPROVED** [review]
Clean Code
All the possible code is extracted out
Ready to merge

**Review by romabulani — APPROVED** [review]
- Code is clean and easily understandable
- Functionality is working as expected
- Suggestion : When I am on another page and not on `Explore` Page, search is not working, you can navigate to explore page first if user is searching on another page or keep the search bar only on explore page

**Pull request merged**